packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md

```diff
@@ -36,7 +36,7 @@ If the `p-fieldset-wrapper` is set to `required="true"`, only the label of the `
 is removed from all wrapped child components, as long as they are Porsche Design System form elements. You should still
 set required on the input of the wrapped form elements to ensure accessibility, and the support of screen readers.
 
-<Playground :markup="requiredMarkup"></Playground>
+<Playground :markup="requiredMarkup" :config="config"></Playground>
```

Comment:
Odd that the config was applied on some playgrounds only.

Reply:
Sorry, what do you mean exactly? `:config` was added to all of the 6 playgrounds. Or do I miss something?